2-environments/modules/env_baseline/networking.tf

```diff
@@ -40,3 +40,52 @@ module "vpc_host_project" {
     application_name = "org-shared-vpc-${var.env}"
   }
 }
+
+
+module "base_shared_vpc_host_project" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 7.0"
+  random_project_id           = "true"
+  impersonate_service_account = var.terraform_service_account
+  name                        = "prj-${var.environment_code}-shared-base"
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  folder_id                   = google_folder.env.id
+  activate_apis = [
+    "compute.googleapis.com",
+    "dns.googleapis.com",
+    "servicenetworking.googleapis.com",
+    "container.googleapis.com",
+    "logging.googleapis.com"
+  ]
+
+  labels = {
+    environment      = var.env
+    application_name = "base-shared-vpc-host-${var.env}"
+  }
+}
+
+module "restricted_shared_vpc_host_project" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 7.0"
+  random_project_id           = "true"
+  impersonate_service_account = var.terraform_service_account
+  name                        = "prj-${var.environment_code}-shared-restricted"
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  folder_id                   = google_folder.env.id
```

Comment:
@daniel-cit just confirming that `data.google_active_folder.networking.id` to `google_folder.env.id` is to align with fig 6.3?

Reply:
Yes, it is to align to fig 6.3. 

form:

```
organization
 |
 |-> "commmon" folder
     |
     |-> "networking" folder
          |
          |-> "org-shared-vpc-prod" project
          |
          |-> "org-shared-vpc-nonprod" project
          |
          |-> "org-shared-vpc-dev" project  
```

to 
 
``` 
organization
 | 
 |-> "prod" folder
 |   |
 |   |-> "prj-p-shared-base" project
 |   |
 |   |-> "prj-p-shared-restricted" project  
 |
 |-> "nonprod" folder 
 |   |
 |   |-> "prj-n-shared-base" project
 |   |
 |   |-> "prj-n-shared-restricted" project  
 |
 |-> "dev" folder
     |
     |-> "prj-d-shared-base" project
     |
     |-> "prj-d-shared-restricted" project  
```

Reply:
Yes, it is to align to fig 6.3. 

form:

```
organization
 |
 |-> "commmon" folder
     |
     |-> "networking" folder
          |
          |-> "org-shared-vpc-prod" project
          |
          |-> "org-shared-vpc-nonprod" project
          |
          |-> "org-shared-vpc-dev" project  
```

to 
 
``` 
organization
 | 
 |-> "prod" folder
 |   |
 |   |-> "prj-p-shared-base" project
 |   |
 |   |-> "prj-p-shared-restricted" project  
 |
 |-> "nonprod" folder 
 |   |
 |   |-> "prj-n-shared-base" project
 |   |
 |   |-> "prj-n-shared-restricted" project  
 |
 |-> "dev" folder
     |
     |-> "prj-d-shared-base" project
     |
     |-> "prj-d-shared-restricted" project  
```